A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/sound/sound.xml

```diff
@@ -863,6 +923,10 @@
         <Sound WwiseData="true" WwiseEvent="aural_100above" NodeName="WIPER_BASE_L" />
         <Sound WwiseData="true" WwiseEvent="aural_retard" />
         <Sound WwiseData="true" WwiseEvent="aural_v1" />
+        <Sound WwiseEvent="FMGCtest" WwiseData="true" NodeName="PEDALS_LEFT" Continuous="false" />
```

Comment:
Is this supposed to be here?

Reply:
Yes, that just defined that the event will have the node in the cockpit. It won't trigger unless you ask it to